guides/client-json-configuration-example.md

```diff
@@ -0,0 +1,181 @@
+# Client JSON Configuration Example
+
+This document provides an example JSON configuration that can be used with `Client.fromConfig()` across the Hiero SDKs. The configuration defines all necessary parameters to connect to and interact with a Hedera network.
+
+## Example Configuration
+
+```json
+{
+  "network": {
+    "35.242.233.154:50211": "3.5.10",
+    "35.240.118.96:50211": "3.5.11",
+    "34.127.45.23:50211": "3.5.12"
+  },
+  "mirrorNetwork": [
+    "hcs.testnet.mirrornode.hedera.com:443"
+  ],
+  "shard": 0,
+  "realm": 0,
+  "grpcDeadline": 10000,
+  "requestTimeout": 120000
+}
+```
+
+## Configuration Parameters
+
+### Network Configuration
+
+- **`network`** (object, required): Maps consensus node addresses to their corresponding version numbers
```

Comment:
I'm not sure if we want to keep `network` as required. It may be the case that a user only has a mirror node endpoint, but they could still use that to query for the address book and get the network configuration